File: trail1/4번의 크기 비교/4-time-comparison.py
A = int(input())
lst = list(map(int, input().split()))
B, C, D, E = lst[0], lst[1], lst[2], lst[3]
print(1 if A > B else 0)
print(1 if A > C else 0)
print(1 if A > D else 0)
print(1 if A > E else 0)

File: trail1/4번의 크기 비교/4-time-comparison.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        int A = int.Parse(Console.ReadLine());
        string[] arr = Console.ReadLine().Split();
        var(B, C, D, E) = (int.Parse(arr[0]), int.Parse(arr[1]), int.Parse(arr[2]), int.Parse(arr[3]));
        Console.WriteLine(A > B ? 1 : 0);
        Console.WriteLine(A > C ? 1 : 0);
        Console.WriteLine(A > D ? 1 : 0);
        Console.WriteLine(A > E ? 1 : 0);
    }
}
